tests/scripts/thread-cert/node.py

```diff
@@ -2479,6 +2479,21 @@ def send_proactive_backbone_notification(self, target: str, mliid: str, ltt: int
         self.send_command(cmd)
         self._expect_done()
 
+    def dns_get_config(self):
+        cmd = f'dns config'
```

Comment:
What about adding a documentation for the return value, or give a return value example? So that, we can know what fields we have the next time we want to use this method.

For example:
https://github.com/openthread/openthread/blob/48b129e74b19bef719cf8388906ef2e7903eae87/tests/scripts/thread-cert/node.py#L854-L874
and
https://github.com/openthread/openthread/blob/48b129e74b19bef719cf8388906ef2e7903eae87/tests/scripts/thread-cert/node.py#L222-L249

Reply:
Changed in new push.